pages/price-feeds/schedule-price-updates/using-scheduler.mdx

```diff
@@ -1,11 +1,240 @@
+
+The configuration file is a YAML file that looks like this:
+```yaml copy
+- alias: A/USD # Arbitrary alias for the price feed. It is used to enhance logging.
+  id: 0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef # id of a price feed, a 32-byte hex string.
+  time_difference: 60 # Time difference threshold (in seconds) to push a newer price feed.
+  price_deviation: 0.5 # The price deviation (%) threshold to push a newer price feed.
+  confidence_ratio: 1 # The confidence/price (%) threshold to push a newer price feed.
+- alias: BNB/USD
+  id: 2f95862b045670cd22bee3114c39763a4a08beeb663b145d283c31d7d1101c4f
+  time_difference: 60
+  price_deviation: 1
+  confidence_ratio: 1
+
+  # Optional block to configure whether this feed can be updated early. If at least one feed meets the
+  # triggering conditions above, all other feeds that meet the early update conditions will be included in
+  # the submitted batch of prices. This logic takes advantage of the fact that adding a feed to a larger
+  # batch of updates incurs a minimal gas cost. All fields below are optional (and interpreted as infinity if omitted)
+  # and have the same semantics as the corresponding fields above.
+  early_update:
+    time_difference: 30
+    price_deviation: 0.1
+    confidence_ratio: 0.5
+- ...
+```
+
+Check out [Price feed IDs](https://pyth.network/developers/price-feed-ids) for the complete list of Price feeds supported by Pyth.
+
+### Running the scheduler
+
+The scheduler communicates with [Hermes](https://docs.pyth.network/price-feeds/how-pyth-works/hermes) to get the most recent price updates. 
+Hermes listens to the Pythnet and Wormhole network to get the latest price updates and provides REST and WebSocket APIs for consumers to fetch the updates. 
+Pyth hosts public endpoints for Hermes; however, getting a private endpoint from one of the Hermes RPC providers is recommended for more reliability. 
+Please refer to [this](https://docs.pyth.network/price-feeds/api-instances-and-providers/hermes) document for more information.
+
+
+To run the scheduler, you need to provide the following parameters:
+- `--pyth-contract-address`: The Pyth contract address where the price updates will be pushed. Check out the [Pyth contract addresses](https://docs.pyth.network/price-feeds/contract-addresses) for the complete list of Pyth contracts.
+- `--price-config-file`: Path to the configuration file.
+- `--price-service-endpoint`: Hermes RPC endpoint.
+- `--mnemonic-file`: Path to the file containing the mnemonic for the account that will push the price updates.
+
+
+If you are using the global installation, you can run the scheduler using the following command:
+
+```bash /npx @pythnetwork/price-pusher/ copy
+npx @pythnetwork/price-pusher start -- evm --endpoint wss://example-rpc.com \
```

Comment:
two comments here:

1) it's going to be annoying to write out a command for each chain *and* each way of running scheduler. Maybe a better approach here would be to only write the command for 1 way of running scheduler (whatever one we think is the favorite), then in the installation instructions above tell the reader to replace the beginning of the command with whatever for the other options

2) These commands written here seem EVM specific, so should these all be under the "EVM" tab below?

Reply:
The idea behind this guide was to introduce them to scheduler. IMO, we can keep the commands as we mentioned in README as it is, as they are self explanatory.